mlflow/tracing/archival/databricks.py

```diff
@@ -0,0 +1,405 @@
+            )
+            -- 6. Main query with optimized structure
+            SELECT
+              rs.trace_id,
+              rs.client_request_id,
+              rs.request_time,
+              rs.state,
+              rs.execution_duration_ms,
+              rs.request,
+              rs.response,
+              rs.trace_metadata,
+              -- Optimized tags processing
+              COALESCE(FROM_JSON(lt.tag_json, 'MAP<STRING, STRING>'), MAP()) AS tags,
+              -- Simplified trace_location construction
+              NAMED_STRUCT(
+                'type',
+                CASE
+                  WHEN rs.experiment_id IS NOT NULL THEN 'mlflow_experiment'
+                  WHEN rs.inference_table_name IS NOT NULL THEN 'inference_table'
+                  ELSE NULL
+                END,
+                'mlflow_experiment',
+                CASE
+                  WHEN rs.experiment_id IS NOT NULL THEN NAMED_STRUCT('experiment_id', rs.experiment_id)
+                  ELSE CAST(NULL AS STRUCT<experiment_id: STRING>)
+                END,
+                'inference_table',
+                CASE
+                  WHEN
+                    rs.inference_table_name IS NOT NULL
+                  THEN
+                    NAMED_STRUCT('full_table_name', rs.inference_table_name)
+                  ELSE CAST(NULL AS STRUCT<full_table_name: STRING>)
+                END
+              ) AS trace_location,
+              COALESCE(aa.assessments, ARRAY()) AS assessments,
+              COALESCE(sa.spans, ARRAY()) AS spans
+            FROM
+              root_spans rs
+                LEFT JOIN latest_tags lt
+                  ON rs.trace_id = lt.trace_id
+                LEFT JOIN assessments_agg aa
+                  ON rs.trace_id = aa.trace_id
+                LEFT JOIN spans_agg sa
+                  ON rs.trace_id = sa.trace_id
+            """
+        
+        spark.sql(query)
+        _logger.info(f"Successfully created trace archival view: {view_name}")
+        
+    except Exception as e:
+        raise MlflowException(f"Failed to create trace archival view {view_name}: {str(e)}") from e
+
+
+def _do_enable_databricks_archival(experiment_id: str, catalog: str, schema: str, table_prefix: str = "trace_logs") -> str:
+    """
+    Enable trace archival by orchestrating the full archival enablement process.
+    
+    Args:
+        experiment_id: The MLflow experiment ID to enable archival for
+        catalog: The Unity Catalog catalog name where tables will be created
+        schema: The Unity Catalog schema name where tables will be created
+        table_prefix: Prefix for the archival view name
+    
+    Returns:
+        The name of the created trace archival view
+        
+    Raises:
+        MlflowException: If any step of the archival process fails
+    """
+    trace_archival_location = f"{catalog}.{schema}.trace_logs_{experiment_id}"
```

Comment:
```suggestion
    trace_archival_location = f"{catalog}.{schema}.{table_prefix}_{experiment_id}"
```

shouldn't this use the given table prefix?